### ISSUE TITLE
[CT-1074] [Feature] Ability to change background color of graph area and node edges (lines)

### ISSUE THREAD

**Issue by noel** [dbt-labs/dbt-core#5707]
### Is this your first time submitting a feature request?

- [X] I have read the [expectations for open source contributors](https://docs.getdbt.com/docs/contributing/oss-expectations)
- [X] I have searched the existing issues, and I could not find an existing issue for this feature
- [X] I am requesting a straightforward extension of existing dbt functionality, rather than a Big Idea better suited to a discussion

### Describe the feature

Now that dbt 1.3 will allow us to change node colors, I thought it would be nice if we could also change the background color of the lineage area.

### Who will this benefit?

users of dbt docs.

### Anything else?

https://getdbt.slack.com/archives/CKHBDNHA6/p1661355317116189

**Comment by sungchun12** [comment]
Note: This will require changes across this repo for setting configs and https://github.com/dbt-labs/dbt-docs for rendering the colors.

**Comment by lostmygithubaccount** [comment]
agreed this would be cool! it's not something the Core team is likely to prioritize for the foreseeable future, so moving out of triage and into the "help wanted" and "good first issue" ice bucket

**Comment by korompaiistvan** [comment]
I would be interested in working on this but I would only be able to get started on it in about a month

**Comment by alzaar** [comment]
Hi! Is anybody working on this ?

**Comment by dbeatty10** [comment]
This is not something that we are likely to prioritize anytime soon, and I'm going to remove the `good_first_issue` and `help_wanted` labels in the meantime.

---

Sorry for any churn here @alzaar! Please let me know if you're looking for a `good_first_issue`, but having trouble finding one. Another place to look would be in the repos for the adapters. Some lists of "good first issues" without pull requests here:
- [dbt-core](https://github.com/dbt-labs/dbt-core/issues?q=is%3Aissue+is%3Aopen+label%3A%22good_first_issue%22+-linked%3Apr)
- [dbt-bigquery](https://github.com/dbt-labs/dbt-bigquery/issues?q=is%3Aissue+is%3Aopen+label%3A%22good_first_issue%22+-linked%3Apr)
- [dbt-redshift](https://github.com/dbt-labs/dbt-redshift/issues?q=is%3Aissue+is%3Aopen+label%3A%22good_first_issue%22+-linked%3Apr)
- [dbt-snowflake](https://github.com/dbt-labs/dbt-snowflake/issues?q=is%3Aissue+is%3Aopen+label%3A%22good_first_issue%22+-linked%3Apr)

**Comment by alzaar** [comment]
Thanks @dbeatty10 for keeping me in the loop. Would it be possible if I have a look at [#5301](https://github.com/dbt-labs/dbt-core/issues/5301) ? Looks like a straightforward change for making initial changes within the repo.

**Comment by fran-at-dribia** [comment]
Does anyone know any workarounds for this? Some script that runs between dbt docs generate and dbt docs serve and just finds-replaces color codes?
